### PR TITLE
Fixed GH-107: synched ./build in starter-template

### LINKED ISSUE #107: Resolve error in make instructions
When trying to run the full make file I get the following error
```
Running errcheck
env GO111MODULE=off /usr/local/go/bin/go get github.com/kisielk/errcheck
errcheck ./server/... 
```
This should be resolved before we create a new release of the plugin.

### PR DESCRIPTION
**Pull request by levb**
- Also removed the golint target from Makefile since it's failing and I
don't see a way to customize its checks

Fixes #107

### CODE REVIEW

File: build/custom.mk [hunk 1]
```diff
@@ -1,9 +1 @@
-## Runs errcheck against all packages.
-.PHONY: errcheck
-errcheck:
```
Comment (hanzei) [inline build/custom.mk]: Can you please elaborate why you are removing this? It runs fine on CI.

Comment (levb) [reply build/custom.mk]: I get this running `make golint` locally, like I said, I did not find a way to change the flags to ignore these. If it works on ci, I am assuming these warnings are disabled somehow?
```
levbrouk@Levs-MacBook-Pro:mattermost-plugin-gitlab (lev-GH-107-errcheck)$ make golint
Running lint
env GO111MODULE=off /Users/levbrouk/go/bin/go get golang.org/x/lint/golint
/Users/levbrouk/gopath/bin/golint -set_exit_status ./...
server/api.go:23:2: don't use ALL_CAPS in Go names; use CamelCase
server/api.go:23:2: exported const API_ERROR_ID_NOT_CONNECTED should have comment (or a comment on this block) or be unexported
server/api.go:24:2: don't use ALL_CAPS in Go names; use CamelCase
server/api.go:27:6: exported type APIErrorResponse should have comment or be unexported
server/api.go:255:6: exported type ConnectedResponse should have comment or be unexported
server/api.go:264:6: exported type GitlabUserRequest should have comment or be unexported
server/api.go:268:6: exported type GitlabUserResponse should have comment or be unexported
server/command.go:12:7: don't use ALL_CAPS in Go names; use CamelCase
```

Comment (hanzei) [reply build/custom.mk]: `golint` is an optional target. We don't run it in CI.

But what you are removing here is `errcheck` which we run in CI and wich also works fine.

Comment (levb) [reply build/custom.mk]: @hanzei ping? This is for 5.16.

**Comment by hanzei** [comment]
@levb Why is this needed for 5.16?

File: build/custom.mk (hunk 1 above)
Comment (levb) [reply build/custom.mk]: Running `errcheck` produces failures. Does it properly support go modules? (running it without, it's failing to locate go-gitlab)

Comment (hanzei) [reply build/custom.mk]: Would you please post the error message here?

**Comment by hanzei** [comment]
@levb Gentle reminder about this PR

**Comment by levb** [comment]
should be ready to go, builds here and locally with go 1.13.3; @hanzei sorry about the errcheck confusion.

**Pull request merged**